api/api.yaml

```diff
@@ -139,6 +139,32 @@ paths:
         '500':
           $ref: '../result/request.yaml#/components/responses/InternalServerError'
 
+  /control/ping:
```

Comment:
I'm presuming here that the API will wait for a certain amount of time (might even be 50ms) before returning.  From the description it is not clear how long the API will wait.  
Also, it is assumed that only one ping packet will be sent?

Reply:
As of now we are just returning if ping is a success or failed, right now it depends on the default time as per the implementation on particular product.

Reply:
- for each src/dst interface pair, ping packet count is 1 and API shall wait for ping response to either be successful or timeout
- we can't provide duration of successful response because RTT for ping may vary based on location of destination interface and number of src/dst interface pairs may be more than 1
- we should however provide an info on how long should it wait before timing out (this could be configurable in future)

@ANISH-GOTTAPU could you please update description accordingly ?

Reply:
We don't want API behavior to be dependent on the implementation of a particular product.
I need to know from the API description the following a) # of ping packets sent, and b) timeout value - ie the time the implementation waits before declaring the request as failed.  We can choose not to expose the timeout value and the description can reflect the current implementation.